concepticondata/conceptlists.tsv

```diff
@@ -186,3 +186,4 @@ Miedema-1985-40	Miedema, J. and Welling, F. I.	1985		40	basic,areal	English	New
 Voorhoeve-1975-40	Voorhoeve, C. L.	1975		40	basic,areal	English	New Guinea		Voorhoeve1975	Voorhoeve1975	A 40 word item list possibly derived from unpublished wordlists of J.C. Anceaux	94-129	
 Dye-1968-120	Dye, W. and Townsend, P. and Townsend, W.	1968		120	basic,areal	English	New Guinea		Dye1968	Dye1968	A 120 item list for a survey of the Sepik languages.	151-152	
 Bromley-1967-111	Bromley, M.	1967		110	basic,areal	English	New Guinea		Bromley1967	Bromley1967	This list builds on [Swadesh's list of 100 items](:ref:Swadesh-1955-100), proposing 11 additional items. Since the original data was collected from various sources, the author's final concept labels vary at times greatly from the concept labels we would expect. The additional items were selected based on the criterion of clear conceptualization and stability. The list was used for a study of Grand Valley Dani languages.	305-307	
+vanderWouw-1974-28	van der Wouw	1974		28	basic,areal	English	New Guinea		vanderWouw1974	vanderWouw1974	This list is a short wordlist designed to demonstrate differences in the dialects of the Asmat languages of New Guinea	5	
```

Comment:
If you mark the PDF field here, you might as well include the PDF, otherwise, leaving it empty is as fine. I'd again end the NOTE in a full stop.